Userland/Libraries/LibJS/Runtime/Intl/PluralRules.cpp

```diff
@@ -14,4 +17,110 @@ PluralRules::PluralRules(Object& prototype)
 {
 }
 
+// 16.5.1 GetOperands ( s ), https://tc39.es/ecma402/#sec-getoperands
+Unicode::PluralOperands get_operands(String const& string)
+{
+    // 1.Let n be ! ToNumber(s).
+    auto number = strtod(string.characters(), nullptr);
```

Comment:
:yakfused: why not check just in case anyway?
```c++
    char* end = nullptr;
    auto number = strtod(string.characters(), &end);
    VERIFY(end == string.characters() + string.length());
```

Reply:
`VERIFY(!*end)` should also work